pages/windows/fc.md

```diff
@@ -0,0 +1,32 @@
+# fc
+
+> Compare the differences between two files or sets of files.
+> Use wildcards (*) to compare sets of files.
```

Comment:
Define "sets of files"?

Reply:
See the 3rd example here:
https://github.com/pxgamer/tldr/blob/master/pages/windows/comp.md

Reply:
Ah, I see